src/quest/QuestParser.cpp

```diff
@@ -0,0 +1,50 @@
+#include "QuestParser.h"
+#include <utility\YiRapidJSONUtility.h>
+
+QuestModel* QuestParser::ParseQuest(const CYIString &filepath)
```

Comment:
Move to QuestModel and rename to FromJson.
Move the Objective parsing into QuestObjective and add FromJson.